examples/matching-engine/src/state.rs

```diff
@@ -63,13 +63,29 @@ pub struct OrderEntry {
     pub order_id: OrderId,
 }
 
+/// The price level is contained in a QueueView
+/// The queue starts with the oldest order to the newest.
+/// When an order is cancelled it is zero. But if that
+/// cancelled order is not the oldest, then it remains
+/// though with a size zero.
 #[derive(View, GraphQLView)]
 #[view(context = "ViewStorageContext")]
 pub struct LevelView {
     pub queue: QueueView<OrderEntry>,
 }
 
-/// The matching engine
+/// The matching engine containing the information.
+/// Meaning of the entries:
+/// * The next_order_number contains the order_id so that
+///   the order_id gets created from 0, to infinity.
+/// * bids containing the bids starting from the best one
+///   (highest proposed price by buyer) to the worse
+/// * asks containing the asks starting from the best one
+///   (smallest asked price by seller) to the worse.
+/// * orders containing the map from the order_id to the
+///   key information
+/// * The account_info containing the list of order_id
+///   by the owner.
```

Comment:
I'd rather put these on the individual fields.

Reply:
Ok, done.